akka-http-core/src/main/scala/akka/http/scaladsl/model/ErrorInfo.scala

```diff
@@ -80,7 +80,10 @@ object EntityStreamException {
  * The limit can also be configured in code, by calling [[HttpEntity#withSizeLimit]]
  * on the entity before materializing its `dataBytes` stream.
  */
-case class EntityStreamSizeException(limit: Long, actualSize: Option[Long] = None) extends RuntimeException {
+final case class EntityStreamSizeException(limit: Long, actualSize: Option[Long] = None) extends RuntimeException {
```

Comment:
technically it's even source incompatible, but I guess we not too strict with this new module